_posts/2020-09-21-fall-2020-meeting-announced.markdown

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: "PyHC Fall 2020 Meeting Dates Announced."
+author: jibarnum
+---
+
+PyHC serves as a community knowledge base for performing heliophysics research in Python, aiming to provide a variety of tutorials, resources, a list of useful packages, general discussion, and advice. Twice yearly, PyHC meets in person. These meetings are open to anyone from individuals already utilizing Python heavily in developing open-source heliophysics packages, as well as those who are just getting into using Python and open source development. 
```

Comment:
I know this paragraph was copy/pasted, but a couple things:
 - The list in the first sentence should read:
> ...provide a variety of **tutorials and resources**, a list of useful packages, general discussion, and advice.   

because of all the items in the list, _a variety of_ only applies to those two.

 - The last sentence isn't worded right. I suggest:
> These meetings are open to anyone, from individuals already utilizing Python heavily in developing open-source heliophysics packages **to** those who are just getting into using Python and open source development.